templates/slurm_stats.sh.epp

```diff
@@ -66,6 +66,17 @@ do
 		echo "${final_pend_line}"
 	fi
 
+        state_string=""
+        states=($(grep ^${p}, ${tfile3} | cut -d',' -f 3 | sort | uniq))
```

Comment:
I don't see where tfile3 is defined in _this_ script.

Reply:
@jakerundall Good catch; I forgot to copy over the creation and clean up of that temp file when I brought these changes over from my internal repo.  Will get a new commit in here and squash this prior one.